src/Components/Home.js

```diff
@@ -32,6 +39,7 @@ function Home() {
           />
         </a>
       ))}
+      {showProgress && <ProgressBar mode="indeterminate" />}
```

Comment:
Should this also go on the Socials component when loading a specific user? I will merge, but another PR could be done to be consistent